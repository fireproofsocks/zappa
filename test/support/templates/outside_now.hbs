These executives have plooked the {{expletive}} out of me
And there's still a long time to go before I've