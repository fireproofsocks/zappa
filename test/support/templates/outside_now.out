These executives have plooked the $$@!@#% out of me
And there's still a long time to go before I've